collagraph/collagraph.py

```diff
@@ -193,6 +199,10 @@ def update_class_component(self, fiber: Fiber):
             fiber.alternate.component = None
             fiber.alternate.component_watcher = None
 
+        if isinstance(fiber.children, dict):
+            component._slots = fiber.children
```

Comment:
If think you should also deal with the situation where `component._slots` was set to something in a previous cycle, and `fiber.children` is now not a dict for whatever reason. If that's the case, I guess you need `component._slots.clear()`?

Or does this never ever happen?

Reply:
Should not happen but better safe than sorry ;)